### PR TITLE
Add include_tags to datadog_monitor

### PR DESCRIPTION
**Pull request by twmartin**
##### SUMMARY

Add `include_tags` option to `datadog_monitor` module.

Closes https://github.com/ansible/ansible/issues/57441

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

- `datadog_monitor`

##### ADDITIONAL INFORMATION

- https://docs.datadoghq.com/api/?lang=python#create-a-monitor

### CODE REVIEW

File: plugins/modules/monitoring/datadog/datadog_monitor.py
Comment (felixfontein) [inline plugins/modules/monitoring/datadog/datadog_monitor.py:141]:
```suggestion
        default: yes
```

File: plugins/modules/monitoring/datadog/datadog_monitor.py
Comment (felixfontein) [inline plugins/modules/monitoring/datadog/datadog_monitor.py:225]:
```suggestion
            include_tags=dict(required=False, default=True, type='bool'),
```
Then the next diff will be cleaner (and not has to touch this line).

File: plugins/modules/monitoring/datadog/datadog_monitor.py
Comment (felixfontein) [inline plugins/modules/monitoring/datadog/datadog_monitor.py:340]: It is already a boolean:
```suggestion
        "include_tags": module.params['include_tags'],
```

**Comment by twmartin** [comment]
Thanks for the feedback, @felixfontein! I've incorporated your suggested changes in d5abcc0 and added the changelog fragment in 98a7b3a.

**Comment by felixfontein** [comment]
Looks good to me. I have no idea on datadog though, and the datadog modules have no tests, so I cannot verify that this change is correct.

@skornehl as the module author and maintainer, can you check this PR?

**Comment by felixfontein** [comment]
BTW, as a future project, there are some sanity check problems which accumulated for the datadog modules over time: https://github.com/ansible-collections/community.general/blob/master/tests/sanity/ignore-2.10.txt#L981-L990 It would be good to fix them, so that new changes are more thoroughly checked. :)

**Comment by gundalow** [comment]
recheck

File: changelogs/fragments/409-datadog-monitor-include-tags.yaml
Comment (Andersson007) [inline changelogs/fragments/409-datadog-monitor-include-tags.yaml:2]:
```suggestion
  - datadog_monitor - add ``include_tags`` option (https://github.com/ansible/ansible/issues/57441).
```

**Comment by Andersson007** [comment]
@skornehl could you please take a look?
@twmartin could you please email the author, there's an email in the module's header.

File: plugins/modules/monitoring/datadog/datadog_monitor.py
Comment (felixfontein) [inline plugins/modules/monitoring/datadog/datadog_monitor.py:141]:
```suggestion
        default: yes
        version_added: 1.3.0
```

**Comment by felixfontein** [comment]
I guess if the author/maintainer doesn't reply in a couple of days, we'll just merge this (once our suggestions are incorporated).

**Comment by Andersson007** [comment]
> I guess if the author/maintainer doesn't reply in a couple of days, we'll just merge this (once our suggestions are incorporated).

agreed

**Comment by felixfontein** [comment]
@twmartin if you accept the suggestions by @Andersson007 and me, I'll merge this.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@twmartin thanks a lot for adding this feature, and sorry that it took so long!
@Andersson007 thanks a lot for reviving this and reviewing!

**Pull request merged**
